### PR TITLE
Update battery unit

### PR DESCRIPTION
**Pull request by tjohn106**
Update battery unit to remove unit warning from Home Assistant in logs

### CODE REVIEW

**Comment by dknowles2** [comment]
This shouldn’t be necessary. Setting the device class to `BATTERY` should automatically set the unit of measurement to the appropriate value. 

I’m also not seeing anything in my logs about this. Are you running my fork of the integration?

**Comment by tjohn106** [comment]
I am using your fork (great edits by the way!).

Here is the log entry that I am getting:

> 2023-01-06 22:22:38.741 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.**XXXXXX**_battery (<class 'custom_components.schlage.sensor.BatterySensor'>) is using native unit of measurement 'None' which is not a valid unit for the device class ('battery') it is using; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.

Also, I was looking through your pyschlage code and saw that there were some additional features for getting and sending codes. Are you building them into your HA integration? If not (or not yet), I was probably going to take a stab at adding those in.

File: custom_components/schlage/sensor.py [hunk 1]
```diff
@@ -32,6 +33,7 @@ class BatterySensor(CoordinatorEntity, SensorEntity):
 
     _attr_device_class = SensorDeviceClass.BATTERY
     _attr_entity_category = EntityCategory.DIAGNOSTIC
+    _attr_unit_of_measurement = PERCENTAGE
```
Comment (dknowles2) [inline custom_components/schlage/sensor.py:36]:
```suggestion
    _attr_native_unit_of_measurement = PERCENTAGE
```

**Comment by dknowles2** [comment]
> Here is the log entry that I am getting:
> 
> > 2023-01-06 22:22:38.741 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.**XXXXXX**_battery (<class 'custom_components.schlage.sensor.BatterySensor'>) is using native unit of measurement 'None' which is not a valid unit for the device class ('battery') it is using; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
> 

Based on that, I think my suggested edit is the correct fix. (Admittedly, my logs are full of garbage from other half-broken integrations I have installed)

> Also, I was looking through your pyschlage code and saw that there were some additional features for getting and sending codes. Are you building them into your HA integration? If not (or not yet), I was probably going to take a stab at adding those in.

I thought about it, but it's not a priority for me. Happy to take PRs though! :-)

**Comment by dknowles2** [comment]
Thanks!

**Pull request merged**